source/tests/pt/test_fitting_stat.py

```diff
@@ -0,0 +1,103 @@
+# SPDX-License-Identifier: LGPL-3.0-or-later
+import unittest
+
+import numpy as np
+
+from deepmd.pt.model.descriptor import (
+    DescrptSeA,
+)
+from deepmd.pt.model.task import (
+    EnergyFittingNet,
+)
+from deepmd.pt.utils.utils import (
+    to_numpy_array,
+    to_torch_tensor,
+)
+
+
+def _make_fake_data_pt(sys_natoms, sys_nframes, avgs, stds):
+    merged_output_stat = []
+    nsys = len(sys_natoms)
+    ndof = len(avgs)
+    for ii in range(nsys):
+        sys_dict = {}
+        tmp_data_f = []
+        tmp_data_a = []
+        for jj in range(ndof):
+            tmp_data_f.append(
+                np.random.normal(  # noqa: NPY002
```

Comment:
why ignore?

Reply:
Ignore to solve warning in `pre-commit.ci`

Reply:
I have set random seed now.

Reply:
Please learn https://docs.astral.sh/ruff/rules/numpy-legacy-random/

Reply:
Thanks. I have deleted the ignore sentence.